src/main/java/nextstep/courses/domain/session/Session.java

```diff
@@ -10,12 +10,12 @@
 public class Session {
     private final SessionId id;
     private final SessionInfo info;
-    private final Enrollment enrollment;
+    private final Enrollments enrollments;
 
-    public Session(SessionId id, SessionInfo info, Enrollment enrollment) {
+    public Session(SessionId id, SessionInfo info, Enrollments enrollments) {
         this.id = id;
         this.info = info;
-        this.enrollment = enrollment;
+        this.enrollments = enrollments;
     }
 
     public void enroll(NsUser user, Payment payment) {
```

Comment:
```suggestion
    public Enrollments enrollments(NsUser user, Payment payment) {
```

Enrollments을 Session의 인스턴스 변수로 구현하지 말고 Session의 정보들을 활용해 Enrollments을 생성하도록 구현하는 것은 어떨까?

Reply:
수강 신청 정보를 갖고 있는 Enrollments는 강의를 의미하는 Session의 필드가 되는 것이 당연하다고 생각하여 이렇게 구현했습니다. 말씀하신 대로 구현하는 것이 객체 사이의 지나친 의존성을 줄일 수 있다는 것에는 공감합니다. 그런데 Enrollments와 Session이 갖는 의미를 생각했을 때 Enrollments가 Session의 필드여야 하지 않냐는 생각이 들더라구요. 제가 객체지향 관점에서 놓친 부분이 있는 거 같은데 의견 부탁 드립니다!